src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts

```diff
@@ -258,6 +265,36 @@ export class FormSubWalletCreationTs extends Vue {
     }
   }
 
+  /**
+   * Get all sub-wallets under the current wallet
+   * @param {string} identifier 
+   * @param {string} address 
+   * @param {string} name 
+   * @return {WalletsModel}
+   */
+  public get currentWallets(): {
+    identifier: string
+    address: string
+    name: string
+  }[] {
+    if (!this.knownWallets || !this.knownWallets.length) {
+      return []
+    }
+
+    // filter wallets to only known wallet names
+    const knownWallets = this.wallets.getWallets(
+      (e) => this.knownWallets.includes(e.getIdentifier()),
+    )
+
+    return [...knownWallets].map(
+      ({identifier, values}) => ({
+        identifier,
+        address: values.get('address'),
+        name: values.get('name'),
+      }),
+    )
+  }
+
```

Comment:
Please do not copy/paste this method. If it is needed multiple times, try to relocate it appropriately.

Reply:
Common data has been stored in vuex